manager/volume.go

```diff
@@ -681,6 +657,62 @@ func (m *VolumeManager) Expand(volumeName string, size int64) (v *longhorn.Volum
 	return v, nil
 }
 
+func (m *VolumeManager) checkAndExpandPVC(namespace string, pvcName string, size int64) (s int64, waitForPVCExpansion bool, err error) {
+	waitForPVCExpansion = false
+
+	pvc, err := m.ds.GetPersistentVolumeClaim(namespace, pvcName)
+	if err != nil {
+		return -1, waitForPVCExpansion, err
```

Comment:
1. The boolean arg should be the 1st arg of the output args.
2. For this function, It's better to directly specify `true` or `false` rather than using a variable in `return`.

Reply:
changed